src/Pact/Types/Command.hs

```diff
@@ -275,6 +275,7 @@ data Payload m c = Payload
   , _pNonce :: !Text
   , _pMeta :: !m
   , _pSigners :: ![Signer]
+  , _pSessionSigner :: Maybe Signer
```

Comment:
This shouldn't go in the payload.

Sessions are a _container-provided trusted value_, whereas Commands are signed by user. 

Thus, the container should be directly instrumenting any Session credentials directly into the environment.

Reply:
Excellent. Things are cleaner now that I've moved it out of the payload and just made it the container's issue.